.github/workflows/release.yaml

```diff
@@ -142,19 +134,14 @@ jobs:
           name: lima-and-qemu.macos-x86
           path: build
 
-      - name: Download MacOS ARM64 dependencies' sources
-        uses: actions/download-artifact@v3
-        with:
-          name: dependencies.sources-arm64
-          path: build
-
-      - name: Download MacOS x86_64 dependencies' sources
+      - name: Download MacOS dependencies' sources
         uses: actions/download-artifact@v3
         with:
-          name: dependencies.sources-x86
+          name: DependenciesSourceCode.zip
           path: build
 
       - name: "Upload to S3"
         run: |
-          aws s3 cp ./build/ s3://${{ secrets.DEPENDENCY_BUCKET_NAME }}/aarch64/ --recursive --exclude "*" --include "lima-and-qemu.macos-aarch64*" --include "dependencies-sources-aarch64*"
-          aws s3 cp ./build/ s3://${{ secrets.DEPENDENCY_BUCKET_NAME }}/x86-64/ --recursive --exclude "*" --include "lima-and-qemu.macos-x86_64*" --include "dependencies-sources-x86_64*"
+          aws s3 cp ./build/ s3://${{ secrets.DEPENDENCY_BUCKET_NAME }}/aarch64/ --recursive --exclude "*" --include "lima-and-qemu.macos-aarch64*"
+          aws s3 cp ./build/ s3://${{ secrets.DEPENDENCY_BUCKET_NAME }}/x86-64/ --recursive --exclude "*" --include "lima-and-qemu.macos-x86_64*" 
```

Comment:
Are these 2 lines related to the purpose of this PR (i.e., `Currently we upload dependency sources for both architectures which is excessive. This change ensures that we only upload 1 version of sources.`)?

Reply:
Previously since we were building the dependency sources for both the architectures we had this line `aws s3 cp ./build/ s3://${{ secrets.DEPENDENCY_BUCKET_NAME }}/aarch64/ --recursive --exclude "*" --include "lima-and-qemu.macos-aarch64*" --include "dependencies-sources-aarch64*`. and `aws s3 cp ./build/ s3://${{ secrets.DEPENDENCY_BUCKET_NAME }}/x86-64/ --recursive --exclude "*" --include "lima-and-qemu.macos-x86_64*" --include "dependencies-sources-x86_64*"`. This put each dependency zip file in a "folder" based on the architecture. There was also a type here `dependencies-sources-aarch64*` , should have been `dependency-sources-${LIMA_ARCH}.${BUILD_TS}.tar.gz. Now since we only have one zip file for dependencies, it can go into `s3://${{ secrets.DEPENDENCY_BUCKET_NAME }}`.